client/src/test/java/sigblockchain/projectred/client/BlockTest.java

```diff
@@ -0,0 +1,78 @@
+package sigblockchain.projectred.client;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import java.math.BigInteger;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class BlockTest {
+
+    private byte[] previousHash = "Previous Hash".getBytes();
+    private byte[] hash = "Block hash".getBytes();
+
+    private byte[][] data =
+            {
+                    {1, 2, 3, 4},
+                    {'H', 'E', 'L', 'L', 'O'},
+                    {5, 6, 7, 8},
+                    {'W', 'O', 'R', 'L', 'D'}
+            };
+
+
+    @Test
+    void testConstructor() {
```

Comment:
All test function names should start in the form of`test<Function Name>`

Reply:
Fixed!